CantoboardFramework/Keyboard/View/KeyCap.swift

```diff
@@ -25,9 +25,9 @@ import UIKit
  but require manual handling in a custom action handler.
 */
 enum SpaceKeyMode: String {
-    case space = "space"
-    case select = "select"
-    case nextPage = "next page"
+    case space = "空格"
```

Comment:
Could we do it in a system locale aware way?

Reply:
iOS shows Chinese labels on Chinese keyboard and English labels on English keyboard, etc. This is a Chinese keyboard, so we should use Chinese.